src/VisualStudio/Core/Def/Packaging/IPackageSearchIOService.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.LanguageServices.Packaging
+{
+    /// <summary>
+    /// Used so we can mock out how the search service does IO for testing purposes.
+    /// </summary>
+    internal interface IPackageSearchIOService
+    {
```

Comment:
Is there a reason this API is synchronous vs being async?  Given that it's fundamentally about I/O async seems more appropriate.